src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Semantic/TestFiles/RazorSemanticTokenInfoServiceTest/GetSemanticTokens_CSharp_Explicit.semantic.txt

```diff
@@ -4,6 +4,7 @@
 0 6 6 namespace name 0 [System]
```

Comment:
I think this whole folder should be deleted, as they won't be relevant baselines any more, with the new test baselines.